tests/test_widget.py

```diff
@@ -2,73 +2,64 @@
 
 import numpy as np
 import pytest
-from cavendish_particle_tracks import ParticleTracksWidget
 
 
-def test_calculate_radius_ui(make_napari_viewer, capsys):
+def test_calculate_radius_ui(cpt_widget, capsys):
     """Test the expected behavior from the expected workflow:
 
     - Add a particle.
     - Add and select three points.
     - Calculate a radius from this.
     - The table should have the correct radius.
     """
-    viewer = make_napari_viewer()
-    viewer.add_image(np.random.random((100, 100)))
-    my_widget = ParticleTracksWidget(viewer)
-
     # need to click "new particle" to add a row to the table
-    my_widget.cb.setCurrentIndex(1)
-    my_widget._on_click_new_particle()
+    cpt_widget.cb.setCurrentIndex(1)
+    cpt_widget._on_click_new_particle()
 
     # add three points to the points layer and select them
-    viewer.add_points([(0, 1), (1, 0), (0, -1)])
-    for layer in viewer.layers:
+    cpt_widget.viewer.add_points([(0, 1), (1, 0), (0, -1)])
```

Comment:
I was thinking of making another version of the fixture that returned both the viewer and the widget, so you'd have something like:

```python
viewer, widget = viewer_and_cpt_widget
```
at the top of each test where you need them both. But feels like a bit overkill since the widget has a reference to its parent viewer.

Reply:
I think the widget is enough, yes.